packages/health/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt

```diff
@@ -59,9 +58,7 @@ class HealthPlugin(private var channel: MethodChannel? = null) : MethodCallHandl
   private var MOVE_MINUTES = "MOVE_MINUTES"
   private var DISTANCE_DELTA = "DISTANCE_DELTA"
   private var WATER = "WATER"
-  private var SLEEP_ASLEEP = "SLEEP_ASLEEP"
```

Comment:
If this is not too match effort I prefer that we will not create braking changes to the health lib. I.e. beep the old sleep end points and add or new SLEEP end point next to it